leap_c/torch/rl/sac.py

```diff
@@ -163,7 +164,9 @@ def __init__(
     def forward(
         self, obs: torch.Tensor, deterministic: bool = False
     ) -> tuple[torch.Tensor, torch.Tensor, dict[str, float]]:
-        """The given observations are passed to the extractor to obtain features.
+        """Actor forward pass.
```

Comment:
Be more expressive, e.g., "Calculates the action to take in the given observation and its log_prob" or sth like this?

Reply:
Yeah I know, but in my opinion it should contain more than what the name of the function already says